README.md

```diff
@@ -60,6 +60,18 @@ or request one from your point of contact at IndyKite.
   "privateKeyPKCS8": "-----BEGIN PRIVATE KEY-----\nM\n-----END PRIVATE KEY-----"
 }
 ```
+A token lifetime is 1h by default.
+You can change this time (from 2 minutes to 24h) by adding a tokenLifetime parameter.
```

Comment:
I would document how to write that lifetime. That is needs to be valid with https://pkg.go.dev/time#ParseDuration function.